explorer/src/server.ts

```diff
@@ -29,15 +30,20 @@ const server = (port: number = DEFAULT_PORT) => {
       },
     }),
   )
+
+  app.use('/api/v1/admin/*', adminAuth)
+  const ADMIN_CONTROLLERS = [controllers.adminNodes]
+  ADMIN_CONTROLLERS.forEach(c => app.use('/api/v1/admin', c))
```

Comment:
not sure if its any more clear, but we could do this if you think it would be better
```suggestion
  const useAdminPath = app.use.bind(undefined,'/api/v1/admin')
  ADMIN_CONTROLLERS.forEach(useAdminPath)
```

Reply:
I think the current approach is more clear as it uses the same pattern of `use` in the file.